src/Controls/samples/Controls.Sample/Pages/Core/ContextFlyoutPage.xaml

```diff
@@ -0,0 +1,128 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="Maui.Controls.Sample.Pages.ContextFlyoutPage">
+
+    <ScrollView>
+        <VerticalStackLayout>
+            <HorizontalStackLayout Spacing="10">
+                <Button Text="Increment by 1 (or right-click me)" Clicked="OnIncrementByOneClicked" FontSize="30" BackgroundColor="DarkGray">
+                    <FlyoutBase.ContextFlyout>
+                        <MenuFlyout>
+                            <MenuFlyoutItem Text="Increment by 10" Clicked="OnIncrementMenuItemClicked" CommandParameter="10"></MenuFlyoutItem>
```

Comment:
Could we include samples using Accelerator keys?

Reply:
Accelerator keys aren't currently wired up. We could look at wiring them up after we're code complete